### DISCUSSION TITLE
layer开启hideOnClose后，在特定场景又需要从后台移除这个层，如何做到？

### DISCUSSION THREAD

**Discussion by zzlatm** [layui/layui#1756] [分享]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.6

### 问题描述

layer开启hideOnClose后，功能运行正常，但在特定场景又希望移除这个层，如何实现？用layer.closeAll()等方法好像不能移除它，还在后台隐藏。

### 业务代码

```auto
let i =layer.open({
    type 		:1,
    id 			:'filterMoreLayer',
   hideOnClose 	:true,
   content 	:content,
    area 		:['580px','100%'],
    shadeClose 	:true,
    offset 		:'r',
   anim 		:'slideLeft',
   //

})
```

### 浏览器

EDGE

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by Sight-wcg** [comment]
根据 ID 查找对应的 DOM 元素，然后移除。
